app/views/notes/show.html.erb

```diff
@@ -117,8 +117,20 @@
       <hr />
     <% end %>
     <% if !@preview %>
-        <%= render partial: "notes/responses" %>
+      <%= render partial: "notes/responses" %>
+      <% if @react %>
+        <!-- some methods aren't available in the controller, so we have to add them to the props object here -->
+        <!-- get_comment_react_props defined in application_helper.rb -->
+        <% comments = get_comment_react_props(@react_props[:comments]) %>
+        <%= react_component("CommentsContainer", { 
```

Comment:
OK here may be where you need to use `<%=raw %>` to avoid escaping markup like `<p>`.